Slim/Utils/Misc.pm

```diff
@@ -84,6 +84,9 @@ my @findBinPaths    = ();
 
 my $MAX_CACHE_ENTRIES = $prefs->get('dbhighmem') ? 512 : 32;
 
+# Sentinel file to skip media folders.
+my $skipsentinel = $prefs->get('skipsentinel');
```

Comment:
Putting this value here would not pick up changes in the sentinel name. That's ok for the scanner, but not in the server's context, where the value can change.

I think reading the value from the prefs should be ok. Yes, it'll be read a few thousand times on a typical scan. But it's accessing an in-memory hash. So that should be pretty quick (compared to the necessary disk access anyway). Just move this line to the `folderFilter()` sub.